enterprise/frontend/src/metabase-enterprise/documents/components/Editor/CommentsMenu/CommentsMenu.tsx

```diff
@@ -39,6 +40,10 @@ export const CommentsMenu = forwardRef<HTMLDivElement, Props>(
     );
     const hasUnresolvedComments = unresolvedCommentsCount > 0;
 
+    if (isWithinIframe()) {
```

Comment:
I wouldn't add this condition inside `CommentsMenu`.
Conditional rendering should be handled by the parent.